app/Models/Foundation/Summit/Registration/PromoCodes/Traits/SpeakersPromoCodeTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php namespace models\summit;
+/**
+ * Copyright 2023 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use Doctrine\Common\Collections\Criteria;
+use Illuminate\Support\Facades\Log;
+use models\exceptions\ValidationException;
+/**
+ * Trait SpeakersPromoCodeTrait
+ * @package models\summit
+ */
+trait SpeakersPromoCodeTrait
+{
+    /**
+     * @ORM\Column(name="Type", type="string")
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @return string
+     */
+    public function getType()
+    {
+        return $this->type;
+    }
+
+    /**
+     * @param string $type
+     */
+    public function setType($type)
+    {
+        $this->type = $type;
+    }
+
+    public function hasOwners(): bool
+    {
+        return count($this->owners) > 0;
```

Comment:
count should be invoked on method ->count()

Reply:
done